### DISCUSSION TITLE
salvare modifiche parametri dispositivo z-wave

### DISCUSSION THREAD

**Discussion by effe-7** [indomus/forum#2720] [HUB: Home Assistant]
Ciao a tutti,
ho la necessità di personalizzare i parametri di un dispositivo z-wave nello specifico un sensore aeotec.

andando nella configurazione del dispositivo modifico i vari parametri **ma non riesco a salvarli**

quando faccio la modifica sul parametro mi mette il pallino del parametro modificato ma poi non ho un salva e se chiudo la configurazione ripristina i parametri di default (direi giustamente senza fare un salva)

grazie

**Comment by indomus** [comment]
> Ciao a tutti, ho la necessità di personalizzare i parametri di un dispositivo z-wave nello specifico un sensore aeotec.
> 
> andando nella configurazione del dispositivo modifico i vari parametri **ma non riesco a salvarli**
> 
> quando faccio la modifica sul parametro mi mette il pallino del parametro modificato ma poi non ho un salva e se chiudo la configurazione ripristina i parametri di default (direi giustamente senza fare un salva)
> 
> grazie

prima di fare il save devi "risvegliare" i dispositivi

**Reply by effe-7** [reply]
il problema è che non ho il "save"
